udata/core/discussions/tasks.py

```diff
@@ -3,7 +3,9 @@
 
 from udata import mail
 from udata.i18n import lazy_gettext as _
-from udata.models import Dataset, Reuse
+from udata.core.dataset.models import Dataset
```

Comment:
why the switch from `udata.models`? Just curious.

Reply:
As much as possible and when I can think about it, I try to remove improt from the `udata.models` facade:
- it's hard to refactor and introspect
- it's one of the circular dependencies sources 
- it's one of the things that increase the test duration

Reply:
👍

Reply:
Looks like something which could benefit of an issue scheduled for `1.3` or something :-)